[docs] Fix typos and grammar in getting started (#17880)
mui-org/material-ui: CHANGELOG.md

- Add color selector (#12053) @mbrookes

> Developers can now dynamically change the theme of the whole documentation.

Reviewer: ```suggestion
You can now dynamically change the theme of the whole documentation site.
```

#### Motivation

> 1. It's a simple pattern to learn. Developers won't need to go back and forth with the documentation to learn the import paths 💭.

Reviewer: ```suggestion
1. It's a simple pattern to learn. You won't need to go back and forth in the documentation to learn the import paths 💭.
```

> 2. Developers bundle size will decrease 🚀.

Reviewer: ```suggestion
2. Your application bundle size will decrease 🚀.
```

- [core] Normalize the API (#7099) @oliviertassinari
Reduce degree of freedom of the API with the color property. That's a tradeoff between correctness and verbosity.

> Developers should be able to recover from that breaking change quite easily as react is going to throw warnings. For instance:

Reviewer: ```suggestion
You should be able to recover from this breaking change quite easily, as React will throw warnings. For instance:
```

###### *Jan 28, 2017*

This is the first public alpha release. It's still work in progress.

> Developers will be able to start giving us feedback with this release.

Reviewer: ```suggestion
You will be able to start giving us feedback with this release.
```
